From c9eec29ea4228de8f90f94e9a2124a6cbc1ebf00 Mon Sep 17 00:00:00 2001
From: Ray Strode <rstrode@redhat.com>
Date: Wed, 14 Oct 2015 14:32:03 -0400
Subject: [PATCH] main: fix typos I introduced

I reworked some of Giovanni's code and introduced
a few typos in commit 6b8d94ae13b2bec1db1bcff4ab4cc67bf63fda9f

https://bugzilla.gnome.org/show_bug.cgi?id=756324
---
 gnome-session/main.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/gnome-session/main.c b/gnome-session/main.c
index 074f53e..4bc8cd0 100644
--- a/gnome-session/main.c
+++ b/gnome-session/main.c
@@ -243,7 +243,7 @@ initialize_gio (void)
         char *use_vfs = NULL;
 
         disable_fuse = g_strdup (g_getenv ("GVFS_DISABLE_FUSE"));
-        use_vfs = g_strdup (g_getenv ("GVFS_USE_VFS"));
+        use_vfs = g_strdup (g_getenv ("GIO_USE_VFS"));
 
         g_setenv ("GVFS_DISABLE_FUSE", "1", TRUE);
         g_setenv ("GIO_USE_VFS", "local", TRUE);
@@ -257,7 +257,7 @@ initialize_gio (void)
         }
 
         if (disable_fuse) {
-                g_setenv ("GIO_DISABLE_FUSE", use_vfs, TRUE);
+                g_setenv ("GVFS_DISABLE_FUSE", use_vfs, TRUE);
                 g_free (disable_fuse);
         } else {
                 g_unsetenv ("GVFS_DISABLE_FUSE");
-- 
2.5.0

